docs/docs-content/vm-management/create-manage-vm/advanced-topics/migrate-vm-kubevirt.md

```diff
@@ -22,6 +22,9 @@ from VMware vSphere to Palette VMO.
 - Only VMs whose operating systems are included under
   [`virt-v2v` supported guest systems](https://libguestfs.org/virt-v2v-support.1.html) can be migrated.
 
+- When using <VersionedLink text="Multus CNI" url="/integrations/packs/?pack=cni-multus" />, the Network Attachment
+  Definition (NAD) used must match the same name assigned to the migration. Otherwise, the migration will fail.
```

Comment:
@karl-cardenas-coding the CLI doesn't prompt the user for a NAD yet, so this is kinda unclear.

A NAD must exist in the migration's target namespace and its name must match the migration name.